### PR TITLE
Add sprint blog post

### PR DESCRIPTION
**Pull request by ChrisWiegman**
## Tasks

- [ ] I've filled out the WP Engine Contributor License Agreement (CLA)

### CODE REVIEW

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+hide_table_of_contents: true
+---
+
+Hello 2023!
+
+It's been a quiet 2 weeks for the Faust team as most of us took time to be with our families for the holidays. As a result we spent the last two weeks looking at a few minor bugs and hope to have a fresh release out soon.
+
+<!--truncate-->
+
+As I write this I'm looking forward to us getting started on our 2023 goals. Over the next two weeks we'll be demonstrating our work on out "admin bar" and Gutenberg features internally as well as building out a roadmap to get them into your hands this quarter.
```
Comment (blakewilson) [inline internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:14]:
```suggestion
As I write this I'm looking forward to us getting started on our 2023 goals. Over the next two weeks we'll be demonstrating our work on our "admin bar" and Gutenberg features internally as well as building out a roadmap to get them into your hands this quarter.
```

**Review by blakewilson — COMMENTED** [review]
@ChrisWiegman Nice! Just one small grammer suggestions

**Comment by ChrisWiegman** [comment]
Good catch, @blakewilson ! I've committed the fix

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+It's been a quiet 2 weeks for the Faust team as most of us took time to be with our families for the holidays. As a result we spent the last two weeks looking at a few minor bugs and hope to have a fresh release out soon.
+
+<!--truncate-->
+
+As I write this I'm looking forward to us getting started on our 2023 goals. Over the next two weeks we'll be demonstrating our work on our "admin bar" and Gutenberg features internally as well as building out a roadmap to get them into your hands this quarter.
+
+Both features, the admin bar and Gutenberg support, promise to really up the capability of WordPress as the best CMS for headless development not just for developers, but for publishers too. From debugging information to easier access to editing content and so much more these features are the start of our push to take Faust from where it is to where we have always envisioned it going.
+
+If you want to see them for yourselves do reach out to the team, we're always ready to show off and get feedback to what we're looking for. You can find us on [Discord](https://discord.gg/J2khkF9XYK) or the [WP Engine Developer's Site](https://developers.wpengine.com/?ref=faustjs).
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:18]:
```suggestion
If you want to see them for yourselves do reach out to the team, we're always ready to show off and get feedback on what we're working on. You can find us on [Discord](https://discord.gg/J2khkF9XYK) or the [WP Engine Developer's Site](https://developers.wpengine.com/?ref=faustjs).
```

Comment (TeresaGobble) [reply internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:18]: I think these changes read more easily.

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Hello 2023!
+
+It's been a quiet 2 weeks for the Faust team as most of us took time to be with our families for the holidays. As a result we spent the last two weeks looking at a few minor bugs and hope to have a fresh release out soon.
+
+<!--truncate-->
+
+As I write this I'm looking forward to us getting started on our 2023 goals. Over the next two weeks we'll be demonstrating our work on our "admin bar" and Gutenberg features internally as well as building out a roadmap to get them into your hands this quarter.
+
+Both features, the admin bar and Gutenberg support, promise to really up the capability of WordPress as the best CMS for headless development not just for developers, but for publishers too. From debugging information to easier access to editing content and so much more these features are the start of our push to take Faust from where it is to where we have always envisioned it going.
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:16]:
```suggestion
Both features, the admin bar and Gutenberg support, promise to really up the capability of WordPress as the best CMS for headless development not just for developers, but for publishers too. From debugging information to easier access to editing content and so much more, these features are the start of our push to take Faust from where it is to where we have always envisioned it going.
```

Comment (TeresaGobble) [reply internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:16]: Comment for clarity.

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Faust Update - January 4, 2023
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:2]: Our naming convention is currently `Sprint ## Update` - is this title an intentional deviation from that?

**Review by TeresaGobble — COMMENTED** [review]
Nicely done Chris! I've added just three small suggestions. Thank you for writing this up for us!

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md (hunk 4 above)
Comment (ChrisWiegman) [reply internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:2]: Yes. I'm changing it for 2023 as "sprint" is confusing to some... not to mention we're back to sprint 1 which could get really confusing later this year.

**Pull request merged**